composer.json

```diff
@@ -93,8 +93,7 @@
         "phpdocumentor/reflection-docblock": "~3.1",
         "phpspec/prophecy": "^1.7",
         "phpunit/phpunit": "^6.5",
-        "pulse00/ffmpeg-bundle": "dev-symfony-4-support",
-        "alchemy/binary-driver": "2.0.0 as 1.6.0",
+        "php-ffmpeg/php-ffmpeg": "dev-master",
```

Comment:
should still be tested against the lowest version we support for 3.4

Reply:
i would say we only support the newest version which is compatible to symfony 4 (not released until now) to make our range of dependencies not bigger then neccessary